src/bTagSF.cc

```diff
@@ -93,6 +93,12 @@ void bTagSF::SetWPset(std::string WPset)
   else if (WPset =="102X_DeepFlavor_V1")
     SetWPset(0.0494, 0.2770, 0.7264);
 
+  else if (WPset == "106X_DeepCSV_V1")
+    SetWPset(0.1208, 0.4168, 0.7665);
+
+  else if (WPset =="106X_DeepFlavor_V1")
+    SetWPset(0.0490, 0.2783, 0.7100);
+
```

Comment:
Assuming (but I might be wrong) that the working points are different for every year I would change these names to "106XUL18" or something similar

Reply:
Good point, I changed the label and added the WP for the other periods

Reply:
+1